media/js/base/mozilla-notification-banner.js

```diff
@@ -16,6 +16,7 @@ if (typeof Mozilla === 'undefined') {
     NotificationBanner.COOKIE_CODE_ID = 'mozilla-notification-banner';
     NotificationBanner.COOKIE_EXPIRATION_DAYS = 21; // default cookie expiry 21 days
     NotificationBanner.COOKIE_INTERACTION_VALUE = 'interacted';
+    NotificationBanner.SAMPLE_RATE = 0.05; // 5% sample rate limit
```

Comment:
Hmm, on second thought - maybe it would just be a bit cleaner. Let me mull it over 😉